gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx2g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
This doesn't seem to be necessary to run Gradle from the command line. I'd rather not include it because this will probably override my Android Studio config. 
If building vanilla K-9 Mail with a freshly installed Android Studio is really a problem I'd rather add a note to the README that tells developers to adjust their Android Studio's config. It also might not be an issue anymore with the recently released Android Studio 3.3.